.github/workflows/nightly-fe-e2e-tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     needs:
       - get-full-docker-image-path
       - build-and-push-featurenet-node-image
-    # to prevent this job to be skipped when on of the parent jobs is skipped
+    # to prevent this job to be skipped when one of the parent jobs is skipped
```

Comment:
"to prevent this job from being skipped when one...", right? same below

Reply:
Right, comment was removed afterall